arrows/core/video_input_image_list.cxx

```diff
@@ -23,182 +23,192 @@
  * ANY DIRECT, INDIRECT, INCIDENTAL, SPECIAL, EXEMPLARY, OR CONSEQUENTIAL
  * DAMAGES (INCLUDING, BUT NOT LIMITED TO, PROCUREMENT OF SUBSTITUTE GOODS OR
  * SERVICES; LOSS OF USE, DATA, OR PROFITS; OR BUSINESS INTERRUPTION) HOWEVER
- * CAUSED AND ON ANY THEORY OF LIABILITY, WHETHER IN CONTRACT, STRICT LIABILITY,
- * OR TORT (INCLUDING NEGLIGENCE OR OTHERWISE) ARISING IN ANY WAY OUT OF THE USE
- * OF THIS SOFTWARE, EVEN IF ADVISED OF THE POSSIBILITY OF SUCH DAMAGE.
+ * CAUSED AND ON ANY THEORY OF LIABILITY, WHETHER IN CONTRACT, STRICT
+ * LIABILITY, OR TORT (INCLUDING NEGLIGENCE OR OTHERWISE) ARISING IN ANY WAY
+ * OUT OF THE USE OF THIS SOFTWARE, EVEN IF ADVISED OF THE POSSIBILITY OF SUCH
+ * DAMAGE.
  */
 
 #include "video_input_image_list.h"
 
-#include <vital/vital_types.h>
-#include <vital/types/timestamp.h>
-#include <vital/types/image_container.h>
-#include <vital/types/image.h>
-#include <vital/types/metadata_traits.h>
 #include <vital/algo/image_io.h>
-#include <vital/exceptions.h>
+
 #include <vital/util/data_stream_reader.h>
 #include <vital/util/tokenize.h>
 
+#include <vital/types/image.h>
+#include <vital/types/image_container.h>
```

Comment:
Not a big deal, but it's not clear what this reordering of includes is based on.  Specifically vital sub-directories (algo, util, types, range).  They are not alphabetical.  I don't see the pattern.